dagfactory/dagbuilder.py

```diff
@@ -795,9 +795,9 @@ def build(self) -> Dict[str, Union[str, DAG]]:
             )
             dag.doc_md = doc_md_callable(**dag_params.get("doc_md_python_arguments", {}))
 
-        # tags parameter introduced in Airflow 1.10.8
-        if version.parse(AIRFLOW_VERSION) >= version.parse("1.10.8"):
-            dag.tags = dag_params.get("tags", None)
+        tags = dag_params.get("tags", [])
+        tags.append("dagfactory")
```

Comment:
Is there any chance `dagfactory` was already added by the end-user? Can we have duplicated tags?

Reply:
Added a check